deploy/osd-pcap-collector/05-pcap-dedicated-admins-ClusterRole.yaml

```diff
@@ -1,13 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This ClusterRole needs to remain to allow customers to run pcaps as dedicated admin.

Reply:
Thank you for pointing this out!  It shouldn't be removed. I don't know why I removed the whole directory..... I will review again when my brain cool down.